sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOSiteVersionPolicyJob.md

```diff
@@ -16,7 +16,7 @@ manager: seanmc
 ## SYNOPSIS
 
 > [!NOTE]
-> This feature is part of the Version history controls Preview. If your tenant is not part of the Preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
+> This feature is part of the version history controls preview. If your tenant is not part of the preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
 
 Cancels further processing of setting version policy job for existing document libraries on the site collection.
```

Comment:
Cancels further processing of version settings update on existing document libraries on the site collection.